### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by epa095**

### CODE REVIEW

**Review by milesgranger — APPROVED** [review]
LGTM! :1st_place_medal: :computer:

File: test_requirements.txt [hunk 1]
```diff
@@ -4,53 +4,53 @@
 #
 #    pip-compile --output-file=test_requirements.txt test_requirements.in
```
Comment (milesgranger) [inline test_requirements.txt]: You now have `--upgrade` in `requirements.txt` but not here.. is this intentional?

Comment (epa095) [reply test_requirements.txt]: Not really, besides that we had that before :-p But yeah, il put it in both. But I think pip-tools removes it in the printout again, so it is an eternal battle keeping it in there :-p

**Comment by epa095** [comment]
This updates pandas from 0.24 to 0.25 as well. Should we just go for it @flikka?

**Pull request merged**